vendor/bat-native-ledger/src/bat/ledger/internal/server/api/api_server.h

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVELEDGER_SERVER_API_API_SERVER_H_
+#define BRAVELEDGER_SERVER_API_API_SERVER_H_
+
+#include <memory>
+
+#include "bat/ledger/ledger.h"
+#include "bat/ledger/internal/server/api/endpoints/get_parameters.h"
+
+namespace bat_ledger {
+class LedgerImpl;
+}
+
+namespace ledger {
+namespace server {
```

Comment:
Not sure that we need the `server` namespace here, seems like we're already conveying that information in the classname below.